security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -17,6 +17,62 @@ To access the latest release, see xref:../../security/compliance_operator/co-man
+
+* When using the Compliance Operator with Cluster Logging Operator (CLO) version 6.0, various rules would fail. This is due to backwards incompatible changes to the CRDs that CLO uses. The Compliance Operator relies on those CRDs to verify logging functionality. The CRDs have been corrected to support the PCI-DSS profiles with CLO. (link:https://issues.redhat.com/browse/OCPBUGS-43229[*OCPBUGS-43229*])
+
+* After installing logging 6.0, users found that the ComplianceCheckResult `ocp4-cis-audit-log-forwarding-enabled` was failing because there was a change in the APIversion of `clusterlogforwarder`. Log collection and forwarding configurations are now specified under the new API, part of the observability.openshift.io API group. (link:https://issues.redhat.com/browse/OCPBUGS-43585[*OCPBUGS-43585*])
+
+* For previous releases of Compliance Operator, the scans would generate an error log for the reconcile loop on the operator pod. With this release, the Compliance Operator controller logic is more stable. (link:https://issues.redhat.com/browse/OCPBUGS-51267[*OCPBUGS-51267*])
+
+* Previously, the rule `file-integrity-exists` would fail during CI runs on `aarch64` clusters. Now that `aarch64` clusters are supported, the rule failure has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-52884[*OCPBUGS-52884*])
+
+* Previously, the rule `file-integrity-notification-enabled` would fail during CI runs on `aarch64` clusters. Now that `aarch64` clusters are supported, this rule failure has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-52885[*OCPBUGS-52885*])
+
+* Before this release of the Compliance Operator, the rule `kubelet-configure-tls-cipher-suites` was failing in CI tests for the API server ciphers, resulting in `E2E-FAILURE` status. The rule was not recognizing new ciphers from RFC 8446. This has been resolved and the rule is now being evaluated correctly. (link:https://issues.redhat.com/browse/OCPBUGS-54212[*OCPBUGS-54212*])
+
+* Previously, the Compliance Operator platform scan would fail and produce the message `failed to parse Ignition config`. With this release, this fault has been resolved and the scan returns either `Compliant` or `Non-Compliant`. link:https://issues.redhat.com/browse/OCPBUGS-54403[*OCPBUGS-54403*])
+
+* Before this release of Compliance Operator, the rule `directory-access-var-log-audit` appears to fail in FedRAMP moderate and high RHCOS profile scans, even after applying the remediation. With this release, the rule correctly completes and detects correct remediation. (link:https://issues.redhat.com/browse/OCPBUGS-54605[*OCPBUGS-54605*])
+
+* Before this release of Compliance Operator, several rules were not platform aware, creating unneeded errors. Now that the rules have been properly ported to other architectures, those rules run correctly. (link:https://issues.redhat.com/browse/OCPBUGS-53041[*OCPBUGS-53041*])
```

Comment:
++

As a result, users may observe some Compliance Check Results reporting `NOT-APPLICABLE` appropriately, depending on the architecture they're using.